packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -1,12 +1,17 @@
 import React, { useEffect, useState, useRef } from 'react';
 import './index.css';
 import type { MouseEvent, FC, ReactNode } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { onLeftPaneCollapseAction, onRightPaneCollapseAction } from '~/store/actions';
+import { DashboardState } from '~/store/state';
 
 export const LEFT_WIDTH_PERCENT = 0.2;
```

Comment:
create a constants file and move these to the new file

Reply:
Done, moved all the constants to separate file.